rasa/telemetry.py

```diff
@@ -72,6 +72,9 @@
     "JENKINS_URL",
     "TEAMCITY_VERSION",
     "TRAVIS",
+    "CODEBUILD_BUILD_ARN",
+    "CODEBUILD_BUILD_ID",
+    "CODEBUILD_BATCH_BUILD_IDENTIFIER",
```

Comment:
why do we add all three? Would one be enough?

Reply:
I wasn't sure which one is always present 😅 the language in the documentation wasn't very specific.

Reply:
at least `CODEBUILD_BUILD_ID` and `CODEBUILD_BUILD_ARN` seem to overlap from the documentation, no?

Reply:
Oh yes, the `ARN` contains the `ID` 🙈 I've already merged `main` into the branch, would it be ok to keep this as it is?

Reply:
sure 👍🏻